sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -337,6 +337,32 @@ object RapidsConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(0)
 
+  val OFF_HEAP_LIMIT_ENABLED = conf("spark.rapids.memory.host.OffHeapLimit.enabled")
+      .doc("Should the off heap limit be enforced or not.")
+      .startupOnly()
+      // This might change as a part of https://github.com/NVIDIA/spark-rapids/issues/8878
+      .internal()
+      .booleanConf
+      .createWithDefault(false)
+
+  val OFF_HEAP_LIMIT_SIZE = conf("spark.rapids.memory.host.OffHeapLimit.size")
+      .doc("The maximum amount of off heap memory that the plugin will use. " +
+          "This includes pinned memory and some overhead memory. If pinned is larger " +
+          "than this - overhead pinned will be truncated.")
+      .startupOnly()
+      .internal() // https://github.com/NVIDIA/spark-rapids/issues/8878 should be replaced with
+      // .commonlyUsed()
+      .bytesConf(ByteUnit.BYTE)
+      .createOptional // The default
+
+  val TASK_OVERHEAD_SIZE = conf("spark.rapids.memory.hostPageable.taskOverhead.size")
```

Comment:
Do we need to mention pageable here, or is it simpler to just call it a per-task overhead size?
```suggestion
  val TASK_OVERHEAD_SIZE = conf("spark.rapids.memory.host.taskOverhead.size")
```